src/Tribe/Promoter/Observer.php

```diff
@@ -64,16 +68,32 @@ public function hook() {
 	 * @since 4.10.1.2
 	 *
 	 * @param $attendee_id
+	 *
+	 * @deprecated TBD
+	 *
 	 */
 	public function notify_rsvp_event( $attendee_id ) {
+		_deprecated_function( __METHOD__, '4.11.5', __CLASS__ . '::notify_ticket_event' );
+
+		$this->notify_ticket_event( $attendee_id );
+	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$ticket_id" missing

<sub>hash: 229eb6e328720de460d3bb08e343ece2</sub>